src/js/components/StatusBar.js

```diff
@@ -42,7 +43,9 @@ export default function StatusBar() {
       >
         <label>{getMessage()}</label>
         <div className="group">
-          {s.ingesting() && <ProgressIndicator percent={value} />}
+          {s.ingesting() && (
+            <ProgressIndicator {...ingestProgressBar.props} percent={value} />
+          )}
```

Comment:
I looks like the progress bar always disappears when ingest is done, but it seems that the parent div (with the existing selector) might not? Like in the case of there being an ingest error.

Reply:
I looked into this. `ingestProgress` is fine for when we expect ingest to succeed. It might even be preferable, because if there is an unexpected error, the test will fail at that point (`ingestProgress` never disappears, even though we are waiting for it to do so). Test authors don't have to worry about "checking that ingest succeeded" because this is taken care of in the app step of `ingestFile()`.

On the other hand for a negative test where we want to test integration of failures, waiting for `ingestProgressBar` is the way to signal the end and then scrape for the error.